diktat-analysis.yml

```diff
@@ -320,5 +320,9 @@
   configuration: {}
 # Checks that property in constructor doesn't contains comment
 - name: KDOC_NO_CONSTRUCTOR_PROPERTY
+  enabled: true
```

Comment:
could you please change the rule. I meant `explicitly`, not `implicitly`

Reply:
Maybe just omit this word? `explicitly` sort of means that you want user to write `constructor` in primary constructor even if it's not needed?

Reply:
yes, good